src/components/KebabMenu.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactElement } from 'react'
```

Comment:
Is this really officially called `KebabMenu`? In material design this is called `OverflowMenu`: https://material.io/components/app-bars-top#anatomy

Reply:
i can rename it the problem is that we have `OverflowMenu` twice then

Reply:
I would also prefer OverflowMenu

Reply:
just import it as something else, or where do we have it as well?

Reply:
In the `KebabMenu` i import the RN-OverflowMenu
adjusted the import
```import { HeaderButtons, OverflowMenu as OverflowMenuRN } from 'react-navigation-header-buttons'```